modules/tinymce/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - New `color_default_foreground` and `color_default_background` options to set the initial default color for the `forecolor` and `backcolor` toolbar buttons and menu items. #TINY-9183
 
 ### Fixed
+- calling getContent method would cause a memory leak. #TINY-9186
```

Comment:
```suggestion
- Parsing media content would cause a memory leak, which for example occurred when using the `getContent` API. #TINY-9186
```
Maybe? Just to show it happening during parsing and only with media (e.g. videos). It also relates it to getting content 🤷